problems/14.Pascal's Triangle.gs

```diff
@@ -0,0 +1,36 @@
+function generate(numRows: number): number[][] {
+
+    let matrix: number[][] = []
+
```

Comment:
Please fix and think about the fact that you are adding random new lines... This is very important!